packages/flutter/lib/src/widgets/animated_size.dart

```diff
@@ -2,7 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
```

Comment:
in all files we should avoid double-blank lines

Reply:
Done